build.project.xml

```diff
@@ -105,6 +105,13 @@
 
     <!-- Execute Drupal updates. -->
     <target name="execute-updates">
+        <drush
+                command="entity:updates"
+                assume="yes"
+                root="${website.drupal.dir}"
+                bin="${drush.bin}"
+                verbose="${drush.verbose}">
+        </drush>
```

Comment:
This should be removed. The entity updates are only intended to be run during development and are too dangerous to run in production environments. They should be replaced by regular updates or post-updates.

Reply:
Aye aye captain.